server/src/assets/game/script-game.js

```diff
@@ -718,9 +772,15 @@ function initialise() {
 	});
 }
 
-$(document).ready(
-	() => initialise()
-);
+function debugDestinationFlags() {
+    $('#trainSelection').hide();
+	$('#destinationsForm').show();
+}
```

Comment:
Function to help with debugging. Hides the train selection, and shows the destination buttons.
**Delete this after you have finished debugging!**